CHANGES.rst

```diff
@@ -4,7 +4,7 @@ Changelog
 4.2 (unreleased)
 ----------------
 
-- Add support for Python 3.7 and 3.8.
+- Add support for Python 3.7 up to 3.9.
```

Comment:
Eh, I'd enumerate them all, in case people want to grep for 3.8 in the changelog.

Reply:
Okay.